libs/protocol-ui/src/stories/atoms/FormLabel.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+import {FormLabel, FormLabelProps} from '../../'
+
+export default {
+    title: 'Components/Atoms/FormLabel',
+    component: FormLabel,
+    argTypes: {
+        label:{
+            control: { type: 'text' },
+        },
+        //color: {control:'color'},
```

Comment:
We can add this back in but this will add a color picker that'll let folks see what this looks like with _any_ color available. We may want to lock it down to only use Chakra/our color tokens, but should be ok to include this for now

Reply:
I see you uncommented that line 12. Thanks!